docs/specs/validation/link-validation.yml

```diff
@@ -412,3 +412,163 @@ outputs:
+    {"message_severity":"suggestion","code":"hard-coded-locale","message":"Link 'http://docs.microsoft.com/tr-tr' contains locale code 'tr-tr'. For localizability, remove 'tr-tr' from links to most Microsoft sites.","file":"file1.md","line":2,"end_line":2,"column":10,"end_column":41}
+
+  file1.json:
+---
+# Validate link preview not set
+inputs:
+  docfx.yml: |
+    markdownValidationRules: rules.json
+  rules.json: |
+    {
+      "links": {
+        "rules": [
+          {
+            "type": "PreserveViewNotSet",
+            "message": "Did you mean to link to a specific version? If so, add &preserve-view=true to the url: {0}",
+            "code": "preserve-view-not-set",
+            "severity": "SUGGESTION",
+            "contentTypes": ["conceptual"]
+          }
+        ]
+      }
+    }
+  file1.md: |
+    [external link](https://docs.microsoft.com/en-us/sql/t-sql/functions/cast-and-convert-transact-sql?view=sql-server-ver15)
+    <a href="https://docs.microsoft.com/en-us/sql/t-sql/functions/cast-and-convert-transact-sql?view=sql-server-ver15">inline html link</a>
+    [relative link](/en-us/sql/t-sql/functions/cast-and-convert-transact-sql?view=sql-server-ver15)
+outputs:
+  .errors.log: |
+    {"message_severity":"suggestion","code":"preserve-view-not-set","message":"Did you mean to link to a specific version? If so, add &preserve-view=true to the url: https://docs.microsoft.com/en-us/sql/t-sql/functions/cast-and-convert-transact-sql?view=sql-server-ver15","file":"file1.md","line":1,"end_line":1,"column":1,"end_column":1}
+    {"message_severity":"suggestion","code":"preserve-view-not-set","message":"Did you mean to link to a specific version? If so, add &preserve-view=true to the url: https://docs.microsoft.com/en-us/sql/t-sql/functions/cast-and-convert-transact-sql?view=sql-server-ver15","file":"file1.md","line":2,"end_line":2,"column":10,"end_column":114}
+    {"message_severity":"suggestion","code":"preserve-view-not-set","message":"Did you mean to link to a specific version? If so, add &preserve-view=true to the url: /en-us/sql/t-sql/functions/cast-and-convert-transact-sql?view=sql-server-ver15","file":"file1.md","line":3,"end_line":3,"column":1,"end_column":1}
+
+  file1.json:
+---
+# Validate link text not set
+inputs:
+  docfx.yml: |
+    markdownValidationRules: rules.json
+  rules.json: |
+    {
+      "links": {
+        "rules": [
+          {
+            "type": "LinkTextPresent",
+            "message": "Missing link text for link '{0}'. Add appropriate text. ",
+            "code": "link-text-missing",
+            "severity": "SUGGESTION",
+            "contentTypes": ["conceptual"]
+          }
+        ]
+      }
+    }
+  file1.md: |
+    [external link](http://docs.microsoft.com/tr-tr)
```

Comment:
according to apex validation, this case "this case: [![](image.png)](https://aka.ms/LintTextValidation)" should pass, 
but not now.